test/bookstore/gke/deploy_secure_template.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: bookstore
+spec:
+  ports:
+    - name: endpoints-http
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+    - name: endpoints-status
+      port: 8090
+      protocol: TCP
+      targetPort: 8090
+  selector:
+    app: bookstore
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: bookstore
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: bookstore
+    spec:
+      volumes:
+      - emptyDir:
+          medium: Memory
+        name: var-log-for-nginx          
+      - emptyDir:
+          medium: Memory
+        name: var-cache-for-nginx
+      - emptyDir:
+          medium: Memory
+        name: home-for-nginx        
+      containers:
+      # [START esp]
+      - name: esp
+        image: $ESP_IMAGE
+        args: [
+          "--http_port=8080",
+          "--backend=127.0.0.1:8081",
+          "--service=${ENDPOINT_SERVICE}",
+          "--rollout_strategy=managed",
+        ]
+      # [END esp]
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        - containerPort: 8090
+          protocol: TCP
+        securityContext:
+          readOnlyRootFilesystem: true
+        volumeMounts:
+          - mountPath: /var/log/nginx
```

Comment:
knative is going to mount a volume at `/var/log`, not `/var/log/nginx`. Your test should probably do the same.

Reply:
ok , will change to /var/log